src/Facilitator.ts

```diff
@@ -0,0 +1,40 @@
+import Config from './Config';
+
+/**
+ * The class defines properties and behaviour of facilitator.
+ */
+export default class Facilitator {
+  private config: Config;
+
+  private chainId: string;
+
+  private dbConnection;
+
+  /**
+   * Facilitator class constructor.
+   *
+   * @param {string} config Config class object.
+   * @param {object} dbConnection DB connection object.
+   * @param {string} chainId Chain identifier to subscribe.
+   */
+  public constructor(config, dbConnection, chainId) {
```

Comment:
is the chainId needed if the facilitator manifest already specifies the chain(s) to facilitate?

Reply:
makes sense 💯 . 
I saw in the ticket that while starting facilitator chainID is specified on command line.
https://github.com/OpenST/facilitator/issues/6
I think that's not needed. Will make the changes.